### PR TITLE
Expose deno.exit() and add test.

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by ry** [comment]
@kt3k Yes - I was being lazy. I've updated it now with a way to check the error code.

**Comment by ry** [comment]
@kt3k ha reading your comment again I realize you weren't asking about the error code at all.

No, I don't think it's so important to prefix the test scripts with numbers. I just did that at the beginning to bootstrap.

**Comment by kt3k** [comment]
@ry OK. I got it.

File: tests/exit_error42.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as deno from "deno";
```
[the remark this answers is not in the document]
Comment (ry) [reply tests/exit_error42.ts]: *shrug* I think this is sufficient. These `tests/*.out` tests are relatively expensive - so I'd rather keep them limited.

**Comment by j-f1** [comment]
Just noticed this API and I’m wondering if it would make sense to instead have a `throw`-based API:

```ts
throw Deno.exit(0)
```

A downside of this is that it can be caught, but it works with TypeScript’s type checker better since it realizes that `throw` breaks out of the current function.

**Pull request merged**